dist/flatpak/com.github.input-leap.input-leaps.yml

```diff
@@ -85,24 +114,24 @@ modules:
           - -Dtests=false
           - -Dportal-tests=false
           - -Dvapi=false
-          - -Dbackends=gtk4
+          - -Dbackend-qt5=auto
+          - -Dbackend-qt6=auto
+          - -Dbackend-gtk4=auto
       sources:
           - type: git
-            url: https://github.com/whot/libportal
-            branch: wip/inputcapture
-
+            url: https://github.com/flatpak/libportal.git
+            branch: 0.8.1
     - name: input-leap
       buildsystem: cmake
       config-opts:
           - -DQT_DEFAULT_MAJOR_VERSION=5
-          - -DINPUTLEAP_BUILD_GUI=OFF
+          - -DINPUTLEAP_BUILD_GUI=ON
           - -DINPUTLEAP_BUILD_TESTS=OFF
           - -DINPUTLEAP_BUILD_INSTALLER=OFF
           - -DINPUTLEAP_BUILD_X11=ON  # FIXME: undefined reference to `XWindowsUtil::mapKeySymToKeyID otherwise
           - -DINPUTLEAP_BUILD_LIBEI=ON
+          - -DINPUTLEAP_DEPLOY_FLATPAK_SCRIPT=ON
```

Comment:
I think you can drop (and `git rm`) the script now given that you start `input-leap` itself. That script was always a PoC anyway.

Reply:
removed script, removed flag, removed if statement in CMakeLists.txt